demo/dynamic_embedding/movielens-1m-keras-notebook/movielens_dynamic_embedding_tutorial.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
<p><code>de.keras.layers.BasicEmbedding</code>  currently has a better alias <code>de.keras.layers.Embedding</code>  which is close to the original keras layer name.</p><p>So it would be better to change all of the BasicEmbedding to Embedding. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/recommenders-addons/pull/315/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>